src/main/java/io/github/perrymant/moneymaker/Main.java

```diff
@@ -1,7 +1,12 @@
 package io.github.perrymant.moneymaker;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class Main {
     public static void main(String[] args) {
-        new Application(new DefaultLogger()).start(args);
+        Balance balance = new Balance();
+        List<Transaction> transactions = new ArrayList<>();
```

Comment:
We shouldn't just pass in an empty array here. The Application still needs to know its transactions in the default case. Previously that was `new TransactionMaker().getTransactions()`